### PR TITLE
cmake: remove counters not used

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -219,7 +219,7 @@ endif(${WITH_XFS})
 
 option(WITH_SPDK "Enable SPDK" OFF)
 if(WITH_SPDK)
-  find_package(dpdk REQUIRED)
+  find_package(dpdk 16.7 REQUIRED)
```
Comment (yuyuyu101) [inline CMakeLists.txt]: do we need to specify version? let user must use submodule to build?

Comment (tchaikov) [inline CMakeLists.txt]: @yuyuyu101 it's a follow up of https://github.com/ceph/ceph/commit/96046f53e9072cee8ce332392635669593335be2#commitcomment-20417415 . 

actually, i tend to put `find_package(dpdk 2.0 REQUIRED)` instead to be consistent with https://github.com/ceph/ceph/pull/12857/commits/1844b729663164248bca17db3e220e036910841a#diff-3b85bae027325c1a3d65ac2b5d316239L32 .

**Comment by optimistyzy** [comment]
currently dpdk is updated to 16.11， why still 16.07？

**Comment by tchaikov** [comment]
@optimistyzy again, please note that use could install older version of dpdk and `Finddpdk.cmake` will find it and use it.

**Comment by tchaikov** [comment]
@yuyuyu101 comments addressed and repushed.

**Comment by yuyuyu101** [comment]
briefly it's ok to me, although I know little in cmake... @liupan1111 does this solve problem?

**Comment by tchaikov** [comment]
@liupan1111 ping?

**Comment by liupan1111** [comment]
@tchaikov @yuyuyu101 , one question, with these changes in this PR, will the core dump in 12854 happen again?

**Comment by yuyuyu101** [comment]
@tchaikov hmm, I think we don't need to have version condition since we have fixed submodule. we only care the submodule version compatible with current dpdk codes

**Comment by tchaikov** [comment]
> with these changes in this PR, will the core dump in 12854 happen again when we use dpdk 16.11?

i don't think so, that's why i was asking you to confirm.

**Comment by liupan1111** [comment]
@tchaikov , ok, let me confirm and let you know

**Comment by liupan1111** [comment]
@tchaikov, I tried your changes, and it works, no assertion failure. Thanks. But I still think it is better to remove version condition. I could give another pr to do some cleanup later.

**Pull request closed without merging**